controllers/articles.controller.js

```diff
@@ -0,0 +1,14 @@
+const selectArticle = require('../models/articles.model')
+
+function getArticle(req, res, next) {
+
+    const { article_id } = req.params
+
+    selectArticle(article_id)
+        .then((article) => {
+            res.status(200).send({ article: article })
```

Comment:
Great! Could use shorthand if you feel comfortable with it

ie. `{ article: article }`
can be shorthand written `{ article}`